content/word-lists/tier-1.md

```diff
@@ -52,21 +72,21 @@ Preferred:
 
 Also acceptable:
 
-* Leader/follower 
+* Leader/follower
 * Parent/child
 
 **Reasoning:**
 
-As the IETF puts it, “Master-slave is an oppressive metaphor that will and should never become fully detached from history.” 
-The word’s origins and historicial use reveal use that is at best chauvinistic and racist, and in almost all cases connotative of ownership. 
-While there is some small ambiguity about the term master, the term slave is unambiguously about the ownership and subjugation of another person, and has been since its inception. 
+As the IETF puts it, “Master-slave is an oppressive metaphor that will and should never become fully detached from history.”
+The word’s origins and historicial use reveal use that is at best chauvinistic and racist, and in almost all cases connotative of ownership.
+While there is some small ambiguity about the term master, the term slave is unambiguously about the ownership and subjugation of another person, and has been since its inception.
 
-The terms master/slave are harmful to Black and people of color contributors and employees. 
-Slavery is a tradition barely 3 generations abolished – there are grandparents alive today who were actual, non-metaphorical slaves. 
-Segregation and Apartheid are even more recent. 
-In accordance with most open source codes of conduct and company handbooks, the mandate of all people in a project is to create a welcoming space, regardless of the level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality. 
+The terms master/slave are harmful to Black and people of color contributors and employees.
```

Comment:
I'm not sure using "employee" in a general world context makes as much sense, additionally people who are paid to contribute to Open Source software are still contributors.

```suggestion
The terms master/slave are harmful to Black and people of color contributors.
```

Reply:
Those are all good changes, Karsten. Thanks.

Reply:
That's a good suggestion, Mark. Your revised wording is much better.

Reply:
@quaid sounds good. I think `indigenous` should be capatilized...proposed fix below.
```suggestion
The master/slave terminology normalizes and perpetuates the historical practice of enslavement.
Additionally, the terms master/slave are directly harmful to Black, Indigenous, and people of color (BIPOC).
```